src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/taglets/TagletWriter.java

```diff
@@ -107,6 +108,16 @@ protected TagletWriter(boolean isFirstSentence) {
      */
     protected abstract Content deprecatedTagOutput(Element element);
 
+    /**
+     * Returns the output for a {@code {@link ...}} or {@code {@linkplain ...}} tag.
+     *
+     * @param element The element that owns the doc comment
+     * @param tag     the tag
+     *
+     * @return the output
+     */
+    protected abstract Content linkTagOutput(Element element, LinkTree tag);
```

Comment:
Finally, a separate method for `{@link}` and `{@linkplain}` tags!

Reply:
:-) Yes, that was one of the goals.